com.unity.perception/Documentation~/Schema/Synthetic_Dataset_Schema.md

```diff
@@ -140,6 +140,7 @@ sensor {
   translation:      <float, float, float> -- Position in meters: (x, y, z) with respect to the ego coordinate system. This is typically fixed during the simulation, but we can allow small variation for domain randomization.
   rotation:         <float, float, float, float> -- Orientation as quaternion: (w, x, y, z) with respect to ego coordinate system. This is typically fixed during the simulation, but we can allow small variation for domain randomization.
   camera_intrinsic: <3x3 float matrix> [optional] -- Intrinsic camera calibration. Empty for sensors that are not cameras.
+  projection:       <string> -- holds the type of projection the camera used for that capture: Options: "perspective" or "orthographic"
```

Comment:
We don't record this per capture do we? So maybe we should just say:
The camera's projection type. Options: "perspective" or "orthographic"

Reply:
Actually we do, I don't know if you remember but we discussed were to put this field and we decided that since it is possible that the user changes the projection type throughout different captures we would place it inside the sensor capture data

Reply:
It also works well like that it is in the same place as the camera_intrinsic matrix for that capture
![image](https://user-images.githubusercontent.com/83719578/124801734-2ad02480-df25-11eb-9b79-2500b3f2710d.png)

Reply:
I don't mind changing to what you said though

Reply:
you're right, old age is catching up with me :)

Reply:
All the other fields there are also per capture but they don't mention "for this capture" in their description so what you said makes more sense regardless